lib/ansible/modules/system/firewalld.py

```diff
@@ -181,6 +201,39 @@
     permanent: yes
     immediate: yes
     state: enabled
+  loop:
+    - ipv4
```

Comment:
This example looks wrong, nowhere is `{{ item }}` used.
The loop should not be added, if `fw_family` is not provided it matches both ipv4 as well as ipv6.
So looping over this is unnecessary and unrecommended.
(Maybe explain this in the `name:` part ?)